automapper/src/main/java/com/valqueries/automapper/ValqueriesUpdateImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.valqueries.automapper;
+
+import io.ran.Property;
+import io.ran.QueryWrapper;
+
+import java.util.function.Function;
+
+public class ValqueriesUpdateImpl<T> implements ValqueriesUpdate<T> {
+	private final T instance;
+	private final QueryWrapper queryWrapper;
+	Property.PropertyValueList propertyValues = new Property.PropertyValueList();
```

Comment:
We need a non property specific property value list, and use that here.

Reply:
https://github.com/persequor-com/valqueries-sql/pull/5/commits/0d489b0cedc021ed20f9d01f8f699b9ac3909bc6